run-tests.sh

```diff
@@ -16,17 +16,8 @@ set -o errexit
 # Quit on unbound symbols
 set -o nounset
 
-# Always bring down docker services
-
-function cleanup() {
-    eval "$(docker-services-cli down --env)"
-}
-trap cleanup EXIT
-
-
 python -m check_manifest --ignore ".*-requirements.txt"
 python -m sphinx.cmd.build -qnN docs docs/_build/html
-eval "$(docker-services-cli up --db ${DB:-postgresql} --search ${SEARCH:-elasticsearch} --cache ${CACHE:-redis} --env)"
```

Comment:
Why should we remove this?

Reply:
My idea was to remove what we are not using. I see no test cases where it is necessary to have a db (etc.) container up and running. Further if we remove those things we could also remove some installation dependencies.